src/lib/consenthandler/consentHandler.js

```diff
@@ -8,6 +10,15 @@ export default class ConsentHandler{
             type: 'iab'
         };
         Object.assign(this.config, option);
+        // Initialize logger with prefix that looks like this
+        // [Pubcid] DEBUG: message...
+
+        prefix.reg(log);
+        prefix.apply(log, {
+            template: '[%n] %l -',
+            nameFormatter: function(name) { return name || 'Pubcid'}
+        });
```

Comment:
Prefix shouldn't be set in this module because it will overwrite caller's own prefix and everything ends up as `Pubcid`.  Maybe in the top level prebid.js since that will only affect the full module.